docs/status.md

```diff
@@ -85,7 +85,7 @@ one of the following tracking labels.
 | dynamic_pad              | no            | revisit      | no             | yes             | no          |
 | dynamic_reshape          | no            | revisit      | infeasible     | yes             | no          |
 | dynamic_slice            | no            | yes*         | yes*           | yes             | no          |
-| dynamic_update_slice     | no            | revisit      | no             | yes             | no          |
+| dynamic_update_slice     | yes           | revisit      | no             | yes             | no          |
```

Comment:
Does Verification still need to be "revisit" given the discussion in this PR?

Reply:
Yes you're right. The constraints C3, C4, C6 in the updated PR that are still not checked should be in type inference, which does not exist yet. I'll create a separate issue to track this, close #554, and set verification to `yes`.

Reply:
Type inference issue #656.